.github/workflows/scanner-mapping-update.yaml

```diff
@@ -22,13 +22,7 @@ jobs:
       uses: google-github-actions/setup-gcloud@v2
 
```

Comment:
(not about this line) Two more things:
1. Do you intend to backport it to `release-4.6` branch?
2. You deleted the PR template and the section about testing I was interested in is missing. How did you check that `.github/workflows/scanner-mapping-update.yaml` still works?

Reply:
1. Yes
2. I was thinking we could do more in-depth testing in the `ROX-26939` branch (PR: https://github.com/stackrox/stackrox/pull/13282), which is the target branch of this PR, and add everything in that PR's testing section (my reasoning was it'd be easier for folks to see all the testing in one place rather than referring to this PR as well)

Reply:
Re 2: deal.